app/src/main/java/com/example/weatherapp/MainActivity.kt

```diff
@@ -54,6 +56,16 @@ class MainActivity : AppCompatActivity() {
 
     override fun onResume() {
         super.onResume()
+        broadcastMapReceiver = object: BroadcastReceiver() {
+            override fun onReceive(context: Context?, intent: Intent?) {
```

Comment:
Why intent here is nullable?
Overall LGTM

Reply:
You are right, it doesn't have to be nullable. It is changed now